### PR TITLE
collect: Add os tag to scollector.collect.* metrics

### PR DESCRIPTION
**Pull request by gbrayut**
I'd also like to add metadata to these metrics but not sure exactly where to do that. I assume they can just use a goroutine to send the metric metadata every hour, or whatever our current design is.

### CODE REVIEW

**Comment by gbrayut** [comment]
Looks like this package is also used by bosun and realog. Those should work fine with this extra tag, or we could default to nil and only set the OS tag for scollector.

**Comment by gbrayut** [comment]
Changed to use an exported collect.Tags variable so that the os tag is only set when used in scollector

**Comment by mjibson** [comment]
The build breaking here was because the go team made a mistake and pushed a bad change to the tools repo. Nothing you did.

**Comment by mjibson** [comment]
LGTM

**Comment by mjibson** [comment]
Metadata uses the metadata package. It's already configured to do the correct thing, just call http://godoc.org/bosun.org/metadata#AddMetricMeta

**Comment by gbrayut** [comment]
Yeah I saw the issue someone filed and linked it in the slack room. Hopefully that doesn't happen too often :-p

I'll try and add metadata to this PR tomorrow. Thanks

**Comment by gbrayut** [comment]
Added metadata. Currently running on ny-winbuild01|ny-rdp01 as a second instance pointing at http://devbosun/

File: cmd/scollector/main.go [hunk 1]
```diff
@@ -266,6 +266,7 @@ func main() {
 	}
 	collectors.DefaultFreq = time.Second * time.Duration(freq)
 	collect.Freq = time.Second * time.Duration(freq)
+	collect.Tags = opentsdb.TagSet{"os": runtime.GOOS}
```
Comment (captncraig) [inline cmd/scollector/main.go]: Do you need to add host in here, or does collect do that for you?

Comment (gbrayut) [inline cmd/scollector/main.go]: I think that is done by the check function. It is working on devbosun so something is adding it for us.

**Comment by captncraig** [comment]
LGTM

**Comment by mjibson** [comment]
lgtm

**Pull request merged**